digdag-ui/model.js

```diff
@@ -297,6 +297,15 @@ export class Model {
     })
   }
 
+  retrySession (session: Session) {
+    const { lastAttempt } = session
+    return this.put('attempts', {
```

Comment:
Yeah that was on purpose, because i wasn't sure which name to use, looking at it now probably this should be set to `wf.name + '-retry'`, what do you think?